contracts/build/OracleController_abi.json

```diff
@@ -34,18 +34,72 @@
     {
         "members": [
             {
-                "name": "key",
+                "name": "id",
                 "offset": 0,
                 "type": "felt"
             },
             {
-                "name": "decimal",
+                "name": "decimals",
                 "offset": 1,
                 "type": "felt"
+            },
+            {
+                "name": "is_abstract_currency",
+                "offset": 2,
+                "type": "felt"
+            },
+            {
+                "name": "starknet_address",
```

Comment:
We probably need logic to upgrade this as new tokens are bridged

Reply:
Never mind, saw you included that. We should emit events when currencies are updated too